### ISSUE TITLE
elevation on cards

### ISSUE THREAD

**Issue by begueradj** [vuetifyjs/vuetify#5170]
### Versions and Environment
Vuetify:
Vue:
Browsers:
OS:

### Previously worked in:
Vuetify:
Vue:

### Reproduction Link

---

### Additional Comments:
Why elevation does not work on Cards?

```
<v-card 
  elevation-22
></v-card>
```

**Comment by begueradj** [comment]
@KaelWD  It does not work as a class either: https://codepen.io/begueradj/pen/WarNxM?&editors=101

**Comment by nekosaur** [comment]
Yes it does https://codepen.io/anon/pen/QZyqqG?editors=1010. There is no class called `elevation-{0}`
